sample_consensus/include/pcl/sample_consensus/impl/lmeds.hpp

```diff
@@ -83,7 +84,7 @@ pcl::LeastMedianSquares<PointT>::computeModel (int debug_verbosity_level)
       continue;
     }
 
-    double d_cur_penalty = 0;
+    double d_cur_penalty;
```

Comment:
I have mixed feelings about this. One one hand, we are declaring without initialization, on another hand, this allows linters to complain about uninitialized value if a branch downstream is not taken.

Reply:
I agree that direct initialization is usually good, but in this case I really think that no initialization is the right thing to do because that value is never read, so it might even be confusing and misleading. Zero would anyway be the wrong value because it means no penalty=perfect model.

Reply:
Replace the block line 111-114 with a ternary operator and construct the value directly there.